content/projects/gsoc/2023/project-ideas/displaying_plugin_health_scores.adoc

```diff
@@ -15,8 +15,8 @@ mentors:
 - "jleon"
 links:
     emailThread: https://community.jenkins.io/t/gsoc2023-project-idea-displaying-of-plugin-health-scores/5318
-//   gitter: "jenkinsci/plugin-installation-manager-cli-tool"
-//   draft: https://docs.google.com/document/d/1s-dLUfU1OK-88bCj-GKaNuFfJQlQNLTWtacKkVMVmHc
+//   gitter: "TBD"
+//   draft: TBD
```

Comment:
@jmMeessen I changed these placeholder links to TBD links per https://github.com/jenkins-infra/jenkins.io/pull/5926#discussion_r1071515000 — I hope they look okay to you.  Thank you.